packages/cashier/src/Components/Form/__tests__/account-transfer-form.spec.js

```diff
@@ -0,0 +1,974 @@
+                account_limits={account_limits}
+                mt5_login_list={mt5_login_list}
+                onMount={onMount}
+                resetConverter={resetConverter}
+                recentTransactionOnMount={recentTransactionOnMount}
+                selected_from={selected_from}
+                selected_to={selected_to}
+                transfer_limit={transfer_limit}
+            />
+        );
+
+        expect(container.querySelector('.account-transfer-form__input')).toBeInTheDocument();
+    });
+
+    it("should show 'Please verify your identity' error if error.code is Fiat2CryptoTransferOverLimit", () => {
+        const accounts_list = [1, 2, 3];
+        const error = {
+            code: 'Fiat2CryptoTransferOverLimit',
+            message: 'testMessage',
+        };
+        const mt5_login_list = [];
+        const onMount = jest.fn();
+        const resetConverter = jest.fn();
+        const recentTransactionOnMount = jest.fn();
+        const transfer_limit = {
+            min: 0,
+            max: 1000,
+        };
+
+        render(
+            <AccountTransferForm
+                accounts_list={accounts_list}
+                account_limits={account_limits}
+                error={error}
+                mt5_login_list={mt5_login_list}
+                onMount={onMount}
+                resetConverter={resetConverter}
+                recentTransactionOnMount={recentTransactionOnMount}
+                selected_from={selected_from}
+                selected_to={selected_to}
+                transfer_limit={transfer_limit}
+            />
+        );
+
+        expect(screen.getByText('Please verify your identity')).toBeInTheDocument();
+    });
+
+    it("should show 'Cashier error' error if error.code is unexpected", () => {
+        const accounts_list = [1, 2, 3];
+        const error = {
+            code: 'testCode',
+            message: 'testMessage',
+        };
+        const mt5_login_list = [];
+        const onMount = jest.fn();
+        const resetConverter = jest.fn();
+        const recentTransactionOnMount = jest.fn();
+        const transfer_limit = {
+            min: 0,
+            max: 1000,
+        };
+
+        render(
+            <AccountTransferForm
+                accounts_list={accounts_list}
+                account_limits={account_limits}
+                error={error}
+                mt5_login_list={mt5_login_list}
+                onMount={onMount}
+                resetConverter={resetConverter}
+                recentTransactionOnMount={recentTransactionOnMount}
+                selected_from={selected_from}
+                selected_to={selected_to}
+                transfer_limit={transfer_limit}
+            />
+        );
+
+        expect(screen.getByText('Cashier Error')).toBeInTheDocument();
+    });
+
+    it('should show <AccountTransferNote /> component', () => {
+        let default_window_inner_width = window.innerWidth;
+        window.innerWidth = 399;
```

Comment:
To test on mobile, you may import `isMobile` then mock its return value.

Reply:
Right, thx for notice!